benchmarks.cc

```diff
@@ -140,6 +162,7 @@ void throughput_producer(unsigned N, Queue* queue, std::atomic<uint64_t>* t0, Ba
     uint64_t expected = 0;
     t0->compare_exchange_strong(expected, __builtin_ia32_rdtsc(), std::memory_order_acq_rel, std::memory_order_relaxed);
 
+    typename Reclaimer::region_guard guard{};
```

Comment:
This guard should be queue-specific and for other queue types it should resolve to an empty structure.